data/ecosystem/vendors.yaml

```diff
@@ -125,6 +125,13 @@
   contact: ''
   oss: false
   commercial: true
+- name: KloudMate
+  distribution: false
+  nativeOTLP: true
+  url: 'https://docs.kloudmate.com/using-opentelemetry-collector'
+  contact: ''
```

Comment:
*contact* is mandatory for new additions to the list. You can provide a github handle or an email address.